packages/d3fc-chart/src/cartesian.js

```diff
@@ -20,9 +20,9 @@ export default (xScale = scaleIdentity(), yScale = scaleIdentity()) => {
     let xOrient = functor('bottom');
     let canvasPlotArea = seriesCanvasMulti();
     let svgPlotArea = seriesSvgMulti();
-    let xAxisStore = store('tickFormat', 'ticks', 'tickArguments', 'tickSize', 'tickSizeInner', 'tickSizeOuter', 'tickValues', 'tickPadding');
+    let xAxisStore = store('tickFormat', 'ticks', 'tickArguments', 'tickSize', 'tickSizeInner', 'tickSizeOuter', 'tickValues', 'tickPadding', 'tickCenterLabel', 'tickLineAlign');
```

Comment:
I'm not sure about this, because `tickCenterLabel` and `tickLineAlign` are not necessarily available on the chosen axis. It works, because `store` only tries to set the property on the axis if it has been given a value for it (i.e. it will fail with a suitable error if you try to set `tickLineAlign` on a non-ordinal axis).

If we don't do this, it makes things a bit harder for the developer - i.e. you'd need to do something like:

```javascript
  fc.chartSvgCartesian({
    xScale: d3.scaleBand(),
    yScale: d3.scaleLinear(),
    xAxis: {
      bottom: scale => fc.axisOrdinalBottom(scale).tickLineAlign('right')
    }
  })
  .xTickSize(10)
  //...
```

instead of 

```javascript
  fc.chartSvgCartesian({
    xScale: d3.scaleBand(),
    yScale: d3.scaleLinear(),
    xAxis: {
      bottom: fc.axisOrdinalBottom
    }
  })
  .xTickLineAlign('right')
  .xTickSize(10)
  //...
```